dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/InternalLogger.kt

```diff
@@ -96,6 +96,16 @@ interface InternalLogger {
         additionalProperties: Map<String, Any?>? = null
     )
 
+    /**
+     * Logs a specific metric from the internal implementation. The metric will leverage the new
+     * added additionalProperties in the
```

Comment:
I'm still not quite clear with the description. Does `internal implementation` mean from SDK-related code only? Also I guess there is a typo in `new added`, and not quite clear how `additionalProperties` will be leveraged. Maybe we can say that metric data (type, values, etc.) will be taken from `additionalProperties`? This way it is more clear imo.

Reply:
the `internal implementation` is present in all the other methods description so I just followed the pattern. And yes for me it means that is meant for internal usage inside the SDK code. I will try to update the last part there ;)